_rules/heading-accessible-name-is-descriptive-fdaadc.md

```diff
@@ -0,0 +1,290 @@
+---
+id: fdaadc
+name: Heading accessible name is descriptive
+rule_type: atomic
+
+description: |
+   This rule checks that the accessible name of a heading describe the topic or purpose of the content.
+   
+accessibility_requirements:
+- wcag20:2.4.6 # Headings and labels (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Dagfinn Rømen
+- Geir Sindre Fossøy
+- Carlos Duarte
+---
+
+## Applicability
+
+This rule applies to any element with the [semantic role](#semantic-role) of `heading` that is [included in the accessibility tree](#included-in-the-accessibility-tree), has an [accessible name](#accessible-name) that is not only [whitespace](#whitespace), and where:
+- the [accessible name](#accessible-name) is different than the [visible](#visible) heading, OR 
```

Comment:
- the [accessible name](#accessible-name) does not [match](#matching-characters) the combined [characters](#character) formed by the [visible](#visible) [descendant text content](#) and [characters](#character) in [images of text](https://www.w3.org/TR/WCAG21/#dfn-images-of-text).